docs/docs/cmd/spo/list/list-get.md

```diff
@@ -30,6 +30,10 @@ m365 spo list get [options]
 
 --8<-- "docs/cmd/_global.md"
 
+## Remarks
+
+When the `properties` option includes values with a `/`, for example: `ListItemAllFields/Id`, an additional `$expand` query parameter will be included on `ListItemAllFields`.
```

Comment:
Let's also add an example where RootFolder/ServerRelativeUrl is retrieved together with the Title and Id

`--properties Title,Id,RootFolder/ServerRelativeUrl`